File: Python/Memory.py
import imp
from Hardware import Hardware

class Memory(Hardware):

    def __init__(self):
        self.freq = 0
        self.memo = 0
        self.supp = "blank"

    def setFreq(self, freq):
        self.freq = freq
    
    def getFreq(self):
        return self.freq

    def setMemo(self, memo):
        self.memo = memo

    def getMemo(self):
        return self.memo

    def setSupp(self, supp):
        self.supp = supp

    def getSupp(self):
        return self.supp

File: PHP/Memory.php
<?php
    
    class Memory extends Hardware
    {
        public $freq;
        public $memo;
        public $supp;
        
        public function __construct($freq = "", $memo = "", $supp = "")
        {
            $this->freq = $freq;
            $this->memo = $memo;
            $this->supp = $supp;
        }

        public function setFreq($freq)
        {
            $this->freq = $freq;
        }
        public function getFreq()
        {
            return $this->freq;
        }


        public function setMemo($memo)
        {
            $this->memo = $memo;
        }
        public function getmemo()
        {
            return $this->memo;
        }


        public function setSupp($supp)
        {
            $this->supp = $supp;
        }
        public function getSupp()
        {
            $this->supp;
        }

        public function __destruct()
        {
            
        }
    }

    
?>